tauri-app/src/scripts/typeshareFix.cjs

```diff
@@ -16,11 +16,9 @@ config_content = config_content.replace(`export interface Network {
 config_content = config_content.replace(`export interface Scenario {
```

Comment:
Is there a reason we don't just want an additional type, rather than replacing the generated one?